spring-cloud-starter-app-broker-logging/build.gradle

```diff
@@ -16,21 +16,7 @@
 
 description = "Spring Cloud App Broker Starter for Logging"
 
-dependencyManagement {
-	imports {
-		mavenBom "org.springframework.boot:spring-boot-dependencies:${springBootVersion}"
-	}
-}
-
 dependencies {
-	compile project(":spring-cloud-starter-app-broker")
-	compile project(":spring-cloud-app-broker-logging")
-}
-
-install {
-	repositories.mavenInstaller {
-		pom.whenConfigured { generatedPom ->
-			generatedPom.packaging = "pom"
```

Comment:
I had to do the following in SCOSB to get it to set the pom packaging on the starters:
```
generatePomFileForMavenJavaPublication {
	pom.packaging = "pom"
}
```